doc/examples/rendering/traceSetsArnold.gfr

```diff
@@ -0,0 +1,348 @@
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["in"], 'noduleLayout:spacing', 2.0 )
+__children["RenderSetup"]["BoxOut"]["in"].setInput( __children["RenderSetup"]["Outputs"]["out"] )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxOut"]["__out"], 'nodule:color', imath.Color3f( 0.240099996, 0.339399993, 0.485000014 ) )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxOut"]["__out"], 'labelPlugValueWidget:renameable', True )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxOut"]["__out"], 'nodule:type', 'GafferUI::StandardNodule' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxOut"]["__out"], 'description', 'The processed output scene.' )
+__children["RenderSetup"]["BoxIn"]["__in"].setInput( __children["RenderSetup"]["in"] )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'nodule:color', imath.Color3f( 0.240099996, 0.339399993, 0.485000014 ) )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'labelPlugValueWidget:renameable', True )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'nodule:type', 'GafferUI::StandardNodule' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'description', 'The input scene' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'plugValueWidget:type', '' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'noduleLayout:spacing', 2.0 )
+__children["RenderSetup"]["__uiPosition"].setValue( imath.V2f( 8.73167801, -126.091125 ) )
+__children["Backdrop"]["title"].setValue( 'Example: Trace Sets (Arnold)' )
+__children["Backdrop"]["description"].setValue( "In this graph we show how to make use of Arnold's trace sets.\n\nArnold allows both objects and rays to be tagged with one or more trace sets. This allows the scene to be compartmentalized, such that affected shading calculations simply dont 'see' objects in other trace sets.\n\nIn this example we will create a mirror that only reflects some of the objects in the scene.\n\n - An object's trace set membership is tagged using Gaffer sets.\n\n - Rays cast from a shader are tagged by inserting a aiTraceSet shader into the shading network before assignment." )
+__children["Backdrop"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -10, 14.9502449 ), imath.V2f( 30, 46.6711006 ) ) )
+__children["Backdrop"]["__uiPosition"].setValue( imath.V2f( -1.96337986, -19.7618332 ) )
+__children["Backdrop1"]["title"].setValue( 'Adding objects to a trace set' )
+__children["Backdrop1"]["description"].setValue( 'Gaffer manages trace sets using the same nodes as normal sets. \n\nThe only difference is that the `render:` prefix is used to denote that it should be converted to a render trace set.\n\nThe SetSpheres node adds the two spheres to their own trace set using the set name `render:spheres`.' )
```

Comment:
### Backdrop text
Gaffer manages trace sets using the same nodes as normal sets. 

The only difference is that the `render:` prefix is used to denote that it should be converted to a render trace set.

~~The~~ **Here, the** ~~SetSpheres~~ **Set_Spheres** node adds the two spheres to their own trace set ~~using the set~~ **with the** name `render:spheres`.

### Comment
Is the `render:` prefix handled specially just for Arnold? If so, we should mention that, and probably be clearer that it's a reserved prefix. Also probably worth mentioning that this conversion only occurs when the scene is translated to Arnold.